### DISCUSSION TITLE
[Question][Environment] kvpn login problem

### DISCUSSION THREAD

**Discussion by Leechangjun1011** [prosyslab-classroom/cs348-information-security#508] [Q&A]
I confused the ID and password for kvpn, so I failed to login many times.
After I found my ID and password, it prohibited the login with message "You can not login with this ID".
How can I solve this problem?
Thanks.

**Comment by RiceBiscuits** [comment]
As I replied to your question in KLMS, you can request for unblocking your account to kvpn administrator (kcloud@kaist.ac.kr).

**Comment by Leechangjun1011** [comment]
Thanks for help.

**Comment by RiceBiscuits** [comment]
I apologize for the mix-up. I confused kcloud and kvpn. I will request for unblock your account to the administrator.

**Comment by RiceBiscuits** [comment]
Does you mean kcloudvpn? or kvpn?

**Comment by Leechangjun1011** [comment]
I mean kcloudvpn. Thanks.
